lib/pages/search/search.dart

```diff
@@ -0,0 +1,63 @@
+import 'package:dartz/dartz.dart' hide State;
+import 'package:fluffychat/app_state/failure.dart';
+import 'package:fluffychat/domain/app_state/contact/get_contacts_success.dart';
+import 'package:fluffychat/mixin/comparable_presentation_contact_mixin.dart';
+import 'package:fluffychat/pages/search/search_view.dart';
+import 'package:fluffychat/pages/new_private_chat/fetch_contacts_controller.dart';
+import 'package:fluffychat/utils/matrix_sdk_extensions/client_stories_extension.dart';
+import 'package:fluffychat/widgets/matrix.dart';
+import 'package:flutter/material.dart';
+import 'package:matrix/matrix.dart';
+import 'package:rxdart/rxdart.dart';
+import 'package:scroll_to_index/scroll_to_index.dart';
+
+class Search extends StatefulWidget {
```

Comment:
too generic name

Reply:
because it needs this name, global search is also make sense, but will have other `global search` function in future (cross products search)